test/component/uk/gov/hmrc/individualsemploymentsapi/controller/v2/LiveEmploymentsControllerSpec.scala

```diff
@@ -0,0 +1,221 @@
+
+    val fromDate = "2017-01-01"
+    val toDate = "2017-09-25"
+
+    scenario("invalid token") {
+      Given("an invalid token")
+      AuthStub.willNotAuthorizePrivilegedAuthToken(authToken, payeEmploymentsScope, retrieveAll = true)
+
+      When("the paye endpoint is invoked")
+      val response = invokeEndpoint(s"$serviceUrl/paye?matchId=$matchId&fromDate=$fromDate&toDate=$toDate")
+
+      Then("the response status should be 401 (unauthorized)")
+      response.code shouldBe UNAUTHORIZED
+      Json.parse(response.body) shouldBe Json.obj(
+        "code"    -> "UNAUTHORIZED",
+        "message" -> "Bearer token is missing or not authorized"
+      )
+    }
+
+    scenario("missing match id") {
+      Given("a valid privileged Auth bearer token")
+      AuthStub.willAuthorizePrivilegedAuthToken(authToken, payeEmploymentsScope, retrieveAll = true)
+
+      When("the paye endpoint is invoked with a missing match id")
+      val response = invokeEndpoint(s"$serviceUrl/paye?fromDate=$fromDate&toDate=$toDate")
+
+      Then("the response status should be 400 (bad request)")
+      response.code shouldBe BAD_REQUEST
+      Json.parse(response.body) shouldBe Json.obj(
+        "code"    -> "INVALID_REQUEST",
+        "message" -> "matchId is required"
+      )
+    }
+
+    scenario("malformed match id") {
+      Given("a valid privileged Auth bearer token")
+      AuthStub.willAuthorizePrivilegedAuthToken(authToken, payeEmploymentsScope, retrieveAll = true)
+
+      When("the paye endpoint is invoked with a malformed match id")
+      val response =
+        invokeEndpoint(s"$serviceUrl/paye?matchId=malformed-match-id-value&fromDate=$fromDate&toDate=$toDate")
+
+      Then("the response status should be 400 (bad request)")
+      response.code shouldBe BAD_REQUEST
+      Json.parse(response.body) shouldBe Json.obj(
+        "code"    -> "INVALID_REQUEST",
+        "message" -> "matchId format is invalid"
+      )
+    }
+
+    scenario("invalid match id") {
+      Given("a valid privileged Auth bearer token")
+      AuthStub.willAuthorizePrivilegedAuthToken(authToken, payeEmploymentsScope, retrieveAll = true)
+
+      When("the paye endpoint is invoked with an invalid match id")
+      val response = invokeEndpoint(s"$serviceUrl/paye?matchId=$matchId&fromDate=$fromDate&toDate=$toDate")
+
+      Then("the response status should be 404 (not found)")
+      response.code shouldBe INTERNAL_SERVER_ERROR
+      response.body shouldBe "{\"statusCode\":500,\"message\":\"NOT_IMPLEMENTED\"}"
+    }
+
+    scenario("valid request to the live paye endpoint implementation") {
+      Given("a valid privileged Auth bearer token")
+      AuthStub.willAuthorizePrivilegedAuthToken(authToken, payeEmploymentsScope, retrieveAll = true)
+
+      And("a valid record in the matching API")
+      IndividualsMatchingApiStub.hasMatchingRecord(matchId, nino)
+
+      And("DES will return employments for the NINO")
+      DesStub.searchEmploymentIncomeForPeriodReturns(
```

Comment:
This is a V2 endpoint and as such we should not reference DES. Can we please remove? We should not need these DES objects for a not yet implemented endpoint.